### PR TITLE
GitHub authentication

### PR DESCRIPTION
**Pull request by stefymojica**

### CODE REVIEW

File: test/MyFirstApiConsume.test.js [hunk 1]
```diff
@@ -85,4 +85,18 @@ describe('First Api Tests', () => {
...
+    const githubUserName = 'stefymojica';
+    const repository = 'workshop-api-testing-js';
+    const response = await axios.get(
+      `${urlBase}/repos/${githubUserName}/${repository}`,
+      { access_token: process.env.ACCESS_TOKEN }
+    );
+
+    expect(response.status).to.equal(StatusCodes.OK);
+    expect(response.data.description).equal('This is a Workshop about API Testing in JavaScript');
+  });
```
Comment (holgiosalos) [inline test/MyFirstApiConsume.test.js:101]: This should go on test/GithubApi.Authentication.test.js file

File: .github/workflows/node.js.yml [hunk 2]
```diff
@@ -12,12 +9,12 @@ jobs:
...
-        cache: 'npm'
-    - run: npm ci
-    - run: npm run build --if-present
-    - name: Run tests
-      run: npm test
+    - run: yarn install
+    - name: Test
+      env:
+        ACCESS_TOKEN: ${{secrets.ACCESS_TOKEN}}
+      run: yarn test
```
Comment (holgiosalos) [inline .github/workflows/node.js.yml:20]: why you are changing npm by yarn?

Comment (stefymojica) [reply .github/workflows/node.js.yml:20]: I was reading about yarn and it seemed cool to implement it in the project and it works the same.

File: test/MyFirstApiConsume.test.js (hunk 1 above)
Comment (stefymojica) [reply test/MyFirstApiConsume.test.js:101]: Fixed, thank you.

File: .github/workflows/node.js.yml (hunk 2 above)
Comment (holgiosalos) [reply .github/workflows/node.js.yml:20]: Ok 👍

**Pull request merged**